wca-guidelines.md

```diff
@@ -72,7 +72,7 @@ To be more informative, each Guideline is classified using one of the following
 - 3d1b+) [CLARIFICATION] Patterns do not need to be present on every face, but only on faces where a distinct color cannot be found. Pieces must not have any features (e.g. textures, pattern irregularities) that significantly distinguish them from similar pieces.
 - 3h+) [CLARIFICATION] Puzzles may be refined internally by sanding, lubrication, or modifications which improve stability (e.g. magnets).
 - 3h++) [EXAMPLE] Examples of enhancements include: new moves are possible, normal moves are impossible, more pieces or faces are visible, colors on the backside of the puzzle are visible, moves are done automatically, or the puzzles have more/different solved states.
-- 3h+++) [EXAMPLE] Carbon fiber and hollowed stickers are considered enhancements.
+- 3h+++) [CLARIFICATION] Carbon fiber stickers with textures are considered enhancements.
```

Comment:
Maybe like this?

Reply:
> Maybe like this?

```suggestion
- 3h+++) [CLARIFICATION] Patterned carbon fiber effect stickers are considered enhancements.
```

- "Carbon fiber effect" because they're not actually carbon fiber.
- "Patterned" not "textures" because I don't think they're necessarily textured, but a printed pattern (I don't have such a puzzle to actually check).